exercises/08_omikuji_project.md

```diff
@@ -0,0 +1,111 @@
+# Exercise 8: Set up and train an Omikuji project
+
+In this lesson, we will set up an Omikuji project.
+[Omikuji](https://github.com/tomtung/omikuji) is an implementation of a family of tree-based machine learning algorithms for extreme multilabel classification. 
+With Omikuji it is possible to build models that emulate (for example) [Parabel](https://doi.org/10.1145/3178876.3185998), [Bonsai](https://arxiv.org/abs/1904.08249), or [AttentionXML](https://arxiv.org/abs/1811.01727) algorithms. 
+The quality of results these algorithms produces is generally very good, even without tuning of hyperparameters. Training can be computationally intensive; by default it will use all available CPU cores in parallel during the training phase. Also large amounts of RAM (several GB) may be required during training, but during use the memory usage is lower.
+
+Note that if you have a local installation of Annif, you will first need to install the 
+[optional Omikuji dependencies](https://github.com/NatLibFi/Annif/wiki/Optional-features-and-dependencies#omikuji-backend).
+
+We will create a project and train it with the same document titles and
+subjects that we used for the TFIDF project.
+
+## 1. Define an Omikuji project in the `projects.cfg` file
+
+Use a text editor to add a new project definition to the end of the
+`projects.cfg` file. 
+
+If you use the `yso-nlf` data set, use the following snippet:
+
+    [yso-omikuji-parabel-en]
+    name=Omikuji Parabel English
+    language=en
+    backend=omikuji
+    analyzer=snowball(english)
+    vocab=yso-en
+
+
+If you use the `stw-zbw` data set, use the following snippet:
+
+    [stw-omikuji-parabel-en]
+    name=Omikuji Parabel English
+    language=en
+    backend=omikuji
+    analyzer=snowball(english)
+    vocab=stw-en
+
+
+These are configurations that emulate Parabel algorithm. All the hyperparameters are left at their default values.
+
+
+## 2. Train the project using sample data
+
+If you use the `yso-nlf` data set, run this command:
+
+    annif train yso-omikuji-parabel-en data-sets/yso-nlf/yso-finna-small.tsv.gz
+
+If you use the `stw-zbw` data set, run this command:
+
+    annif train stw-omikuji-parabel-en data-sets/stw-zbw/stw-econbiz-small.tsv.gz
+
+## 3. Test the project on sample text
+
+Once training is completed, we can try the model on some example sentence.
+
+If you use the `yso-nlf` data set, run this command:
+
+    echo "library for efficient learning of word representations and sentence classification" | annif suggest yso-omikuji-parabel-en
```

Comment:
The example sentence was taken from fastText docs; maybe use something different here?